VideoWeb/VideoWeb/ClientApp/src/app/app.component.spec.ts

```diff
@@ -110,15 +108,6 @@ describe('AppComponent', () => {
         expect(connectionStatusServiceSpy.start).toHaveBeenCalled();
     }));
 
-    it('should start event service if authenticated oninit', fakeAsync(() => {
```

Comment:
When do you propose to start the connection to EventHub if not once authenticated?

Reply:
The constructor of EventsHubService now does the building of the HubConnection which raises an event when it is set up which is picked up by EventsService; EventsService then starts the connection. The service is never created until after the user has logged in and has been redirected from the MS login page, so it cannot start until the user is authenticated. The start method also checks if the user is authenticated before actually starting the HubConnection.